anomalib/core/metrics/training_stats.py

```diff
@@ -0,0 +1,52 @@
+"""Module that computes the parameters of the normal data distribution of the training set."""
+from typing import Optional, Tuple
+
+import torch
+from torch import Tensor
+from torchmetrics import Metric
+
+
+class TrainingStats(Metric):
```

Comment:
`TrainingDistribution`, `AnomalyScoreDistribution`, `NormalDataDistribution`, `TrainingSetStatistics`?

Reply:
`TrainingDistribution` or `NormalDataDistribution` sound more descriptive to me. What do you think @djdameln  and @ashwinvaidya17

Reply:
Technically, the class could also be applied to the validation set or some collection of abnormal data. So maybe I would vote for `AnomalyScoreDistribution`, which covers all cases.

Reply:
Because essentially the class parameterizes the distribution of the anomaly scores of some collection of data. The source of this collection could be any dataset or subset, and the type of labels in the dataset could be normal, anomalous or mixed.

Reply:
yes, good point! In this case, I'd also vote for `AnomalyScoreDistribution`